src/test/java/org/example/marketplacebackend/TestAccountEndpoints.java

```diff
@@ -0,0 +1,54 @@
+package org.example.marketplacebackend;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.example.marketplacebackend.model.Account;
+import org.example.marketplacebackend.repository.AccountRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import java.sql.Date;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class TestAccountEndpoints {
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Test
+  void testRegisterUser() throws Exception {
+    Account account = new Account();
+    account.setUsername("test");
+    account.setEmail("test@mail.com");
+    account.setPassword("test123");
+    account.setFirst_name("test1");
+    account.setLast_name("testsson");
+    account.setDate_of_birth(Date.valueOf("1993-03-11"));
+
+    ObjectMapper objectMapper = new ObjectMapper();
+    String json = objectMapper.writeValueAsString(account);
+
+    ResultActions createUser = mockMvc.perform(post("/v1/accounts/register")
+        .contentType(MediaType.APPLICATION_JSON)
+        .content(json));
+
+    createUser.andExpect(status().isCreated());
```

Comment:
Add a cleanup method either by using `@AfterEach` or `@AfterAll`. Make this method delete database entries added in tests. This method will ensure that the testing environment is clean.

Reply:
> Add a cleanup method either by using `@AfterEach` or `@AfterAll`. Make this method delete database entries added in tests. This method will ensure that the testing environment is clean.

Tried this with Jakob it does not work because MockMVC auto configuration